### PR TITLE
Fixed ZKEnvelope leak

### PR DESCRIPTION
**Pull request by pilot34**
Hello. I think there was something strange with `env` in `ZKEnvelope`. There was either a leak or a zombie.

### CODE REVIEW

File: zkSforce/zkEnvelope.m [hunk 1]
```diff
@@ -33,10 +33,15 @@ @implementation ZKEnvelope
...
 - (void)start:(NSString *)primaryNamespceUri {
 	[env release];
-	env = [NSMutableString stringWithFormat:@"<s:Envelope xmlns:s='http://schemas.xmlsoap.org/soap/envelope/' xmlns:x='http://www.w3.org/2001/XMLSchema-instance' xmlns='%@'>", primaryNamespceUri];
+	env = [[NSMutableString alloc] initWithFormat:@"<s:Envelope xmlns:s='http://schemas.xmlsoap.org/soap/envelope/' xmlns:x='http://www.w3.org/2001/XMLSchema-instance' xmlns='%@'>", primaryNamespceUri];
 	state = inEnvelope;
 }
 
+- (void)dealloc {
+    [super dealloc];
+    [env release];
```
Comment (superfell) [inline zkSforce/zkEnvelope.m]: these 2 should be the other way around.

Comment (pilot34) [inline zkSforce/zkEnvelope.m]: sure

**Comment by superfell** [comment]
Thanks!

**Comment by superfell** [comment]
BTW, are you using ZKEnvelope in some different way to have run into this?

**Comment by pilot34** [comment]
No, only from public methods of ZKSforceClient. I just profiled my app with instruments and found 2 memory leaks by stacktrace: this from zkEnvelope and second from ZKNamespacedName, that you've already fixed earlier.

**Pull request merged**